### DISCUSSION TITLE
I want to dynamic set proxy-buffering parameter on the route, so fit the SSE situation, can you help me ?

### DISCUSSION THREAD

**Discussion by zzqCh** [Kong/kong#11633] [Help]
I want to dynamic set proxy-buffering  parameter on the route, not custom the nginx configuration.

proxy_buffering is : 
https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_buffering

SSE means  server send event

**Comment by StarlightIbuki** [comment]
Do you mean you want to conditionally buffer the proxy response? Maybe you could try the [proxy-cache](https://docs.konghq.com/hub/kong-inc/proxy-cache/) plugin.
